README.md

```diff
@@ -122,20 +122,21 @@ Comments
 **Example:**
 ```objc
 /*
- * This is a multi-line comment. The opening and closing markers are on their
- * own lines, and each other line is preceded by a * that is indented by one
- * space.
- *
- * This is a new paragraph in the same block comment.
+ This is a multi-line comment. The opening and closing markers are on their
+ own lines, and each other line is preceded by a * that is indented by one
+ space.
+
+ This is a new paragraph in the same block comment.
  */
 
 stop(); // Hammer-time!
 
 // this is a very brief comment.
 ```
 
-* Use [doxygen](http://www.stack.nl/~dimitri/doxygen/manual/docblocks.html)-style comments for all the headers. Make
-sure to use the available markup tags like `@param`, `@return`, etc. The `///` form is preferred for single line
+* Document all methods and properties in the headers using Xcode8's documentation style. You can select a property
```

Comment:
Xcode8's 👉 Xcode 8’s

I.e. space between Xcode and 8 and typographic apostrophe.